components/Heading/Heading.js

```diff
@@ -1,33 +1,67 @@
-import React from 'react';
-import { string, number, oneOfType, bool, node, oneOf } from 'prop-types';
+import React, { Component } from 'react';
+import { string, number, oneOfType, bool, oneOf } from 'prop-types';
 import classNames from 'classnames';
+import kebabCase from 'lodash/kebabCase';
+import ScreenReaderOnly from 'components/ScreenReaderOnly/ScreenReaderOnly';
+import LinkIcon from 'static/images/icons/FontAwesome/link-solid.svg';
 import styles from './Heading.css';
 
-Heading.propTypes = {
-  className: string,
-  id: oneOfType([string, number]), // reference for scroll anchors
-  children: node.isRequired,
-  hasHeadingLines: bool,
-  theme: oneOf(['gray', 'secondary', 'white']),
-};
+class Heading extends Component {
+  static propTypes = {
+    className: string,
+    id: oneOfType([string, number]), // reference for scroll anchors
+    customAnchorClass: string,
+    text: string.isRequired,
+    hasHeadingLines: bool,
+    hasHashLink: bool,
+    headingLevel: number,
+    theme: oneOf(['gray', 'secondary', 'white']),
+  };
 
-Heading.defaultProps = {
-  className: undefined,
-  id: '',
-  hasHeadingLines: true,
-  theme: 'gray',
-};
+  static defaultProps = {
+    className: undefined,
+    id: '',
+    customAnchorClass: 'anchorMargin',
+    hasHeadingLines: false,
+    hasHashLink: true,
+    headingLevel: 2,
+    theme: 'secondary',
+  };
 
-function Heading({ children, className, hasHeadingLines, id, theme }) {
-  const classes = classNames(className, styles.Heading, styles[theme], {
-    [`${styles.headingLines}`]: hasHeadingLines,
-  });
+  render() {
+    const { props } = this;
+    const anchorId = kebabCase(props.text);
+    const classes = classNames(styles[props.theme], {
+      [`${styles.Heading} ${props.className}`]: !props.hasHeadingLines,
+      [`${styles.headingOurMission}`]: props.className === 'headingOurMission',
```

Comment:
You'll still wanna kill this line and pass `headingOurMission` into that specific usage of `<Heading>` instead of in this component.